n4js-libs/packages/n4js-runtime/src/n4js/ReflectionBuilder.n4js

```diff
@@ -0,0 +1,437 @@
+/*
+ * Copyright (c) 2020 NumberFour AG.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   NumberFour AG - Initial API and implementation
+ */
+
+import {N4Type, N4Classifier, N4Class, N4Interface, N4EnumType, N4Annotation, N4DataField, N4Accessor, N4Method, N4Member} from 'N4BuiltInTypes';
```

Comment:
Or maybe a namespace import? Would also make the "cast from N4Class to N4Class" situation a bit more understandable.

Reply:
My first though was "cool" and I applied the change. But I had to change a lot of places and since this would be undone when we move the Builtin types to n4js-runtime, I think I rather leave it like this. Still, I added a comment to the N4Class cast.